go/vt/vtgate/vstream_manager.go

```diff
@@ -575,15 +578,24 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 			})
 		}()
 
-		log.Infof("Starting to vstream from %s", tablet.Alias.String())
+		var options *binlogdatapb.VStreamOptions
+		const SidecardHeartbeatTableName = "heartbeat"
+		if vs.flags.GetStreamKeyspaceHeartbeats() {
+			options = &binlogdatapb.VStreamOptions{
+				InternalTables: []string{SidecardHeartbeatTableName},
+			}
+		}
+
 		// Safe to access sgtid.Gtid here (because it can't change until streaming begins).
 		req := &binlogdatapb.VStreamRequest{
 			Target:       target,
 			Position:     sgtid.Gtid,
 			Filter:       vs.filter,
 			TableLastPKs: sgtid.TablePKs,
+			Options:      options,
 		}
 		var vstreamCreatedOnce sync.Once
+		log.Infof("Starting to vstream from %s, with req %+v", tablet.Alias.String(), req)
```

Comment:
I'd like to see us use `topoproto.TabletAliasString(tablet.Alias)` everywhere as it makes the logging/errors uniform and the value lines up directly with what people see in e.g. `vtctldclient GetTablets` output.

Reply:
fixed